README.md

```diff
@@ -253,10 +280,13 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 |Showcase | Description |
 | ------- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | <a href="https://www.abstrakt.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184916241-46cc2949-d749-4695-9813-05569b88af99.png' alt="Abstrakt_freemium_illustrations" /></a> | [Abstrakt](https://www.abstrakt.design/)  <br><br> ✨ Freemium <br><br>Format <br> • PNG<br>• SVG<br><br> ✌️Author <br> Abstrakt.design                                                                                                                |
+| <a href="https://animalz.design/"><img width='700' src='https://user-images.githubusercontent.com/34191321/194711955-ef309dbf-5620-4737-ab62-ff0aabcdd9f0.png' alt="Animalz_freemium_illustrations" /></a> | [Animalz](https://animalz.design/)  <br><br> ✨ Freemium  <br><br>Format <br> • Sketch<br>• Adobe Photoshop<br>• Adobe Illustrator<br>• Adobe XD<br>• Figma<br>• InVision Studio<br>• Blender 3D<br><br> ✌️Author <br> ThreeDee                       |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195574700-bffeaa34-052f-4886-9daa-2d4daf7d43ac.png`